packages/devops/scripts/lambda/actions/spin_up_tupaia_deployment.py

```diff
@@ -96,18 +97,15 @@ def spin_up_tupaia_deployment(event):
         security_group_code=security_group_code,
     )
 
-    # clone db instance
-    # do this after the server has started because it will take a while to run its startup script, so
-    # we might as well be cloning the db instance at the same time, so long is it is available before
+    # create db instance from a snapshot
+    # do this after the server has started because it will take a while to populate the db from the snapshot, 
+    # so we might as well be cloning the db instance at the same time, so long is it is available before
     # the server first tries to connect
-    deployment_type='tupaia'
-    clone_instance(
-        deployment_type,
-        clone_db_from,
+    create_db_instance_from_snapshot(
         deployment_name,
-        instance_type,
-        extra_tags=extra_tags + [{ 'Key': 'DeploymentComponent', 'Value': 'db' }],
```

Comment:
We can remove all the other `DeploymentComponent` stuff too (some should have todos on it)